src/main/java/com/google/firebase/auth/FirebaseUserManager.java

```diff
@@ -449,27 +367,12 @@ ListSamlProviderConfigsResponse listSamlProviderConfigs(int maxResults, String p
 
   void deleteOidcProviderConfig(String providerId) throws FirebaseAuthException {
     GenericUrl url = new GenericUrl(idpConfigMgtBaseUrl + getOidcUrlSuffix(providerId));
-    sendRequest("DELETE", url, null, GenericJson.class);
+    httpClient.sendRequest("DELETE", url, null, GenericJson.class);
   }
 
   void deleteSamlProviderConfig(String providerId) throws FirebaseAuthException {
     GenericUrl url = new GenericUrl(idpConfigMgtBaseUrl + getSamlUrlSuffix(providerId));
-    sendRequest("DELETE", url, null, GenericJson.class);
-  }
-
-  private static Set<String> generateMask(Map<String, Object> properties) {
-    ImmutableSortedSet.Builder<String> maskBuilder = ImmutableSortedSet.naturalOrder();
-    for (Map.Entry<String, Object> entry : properties.entrySet()) {
-      if (entry.getValue() instanceof Map) {
-        Set<String> childMask = generateMask((Map<String, Object>) entry.getValue());
-        for (String childProperty : childMask) {
-          maskBuilder.add(entry.getKey() + "." + childProperty);
-        }
-      } else {
-        maskBuilder.add(entry.getKey());
-      }
-    }
-    return maskBuilder.build();
+    httpClient.sendRequest("DELETE", url, null, GenericJson.class);
   }
 
   private static String getTenantUrlSuffix(String tenantId) {
```

Comment:
Can `getTenantUrlSuffix` be removed now that the tenant-related methods have been removed?

Reply:
Done